README.md

```diff
@@ -1,9 +1,9 @@
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/airbnb/javascript?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-
-# Airbnb JavaScript Style Guide() {
+# Hudl JavaScript Style Guide() {
```

Comment:
Yes, sorry, my company forked this and somehow my first PR attempt went to this repo, so I just kept it up. Didn't see this. I'll change it.